snowpack.config.js

```diff
@@ -0,0 +1,21 @@
+// Snowpack Configuration File
```

Comment:
can we use https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chrome/index.d.ts?

Reply:
This is already the dev dependency, check line 27 of package.json. It's installed by `npm install --save-dev @types/chrome`, refer to the [usage](https://github.com/DefinitelyTyped/DefinitelyTyped#what-are-declaration-files-and-how-do-i-get-them) of those declaration files